### PR TITLE
Append tokens from uniswap default list

### PR DESCRIPTION
**Pull request by hungdoansy**
- Append tokens from uniswap default list
- This PR also removes tokens that are not found on CoinMarketCap

### CODE REVIEW

File: tokenLists/ethereum.tokenlist.json [hunk 1]
```diff
@@ -49,15 +46,6 @@
...
         },
-        {
-            "chainId": 1,
-            "address": "0x0ccD5DD52Dee42B171a623478e5261C1eaaE092A",
-            "_scan": "https://etherscan.io/token/0x0ccD5DD52Dee42B171a623478e5261C1eaaE092A",
-            "symbol": "DFM",
-            "name": "DFM",
-            "decimals": 18,
-            "logoURI": "https://mycoldwalletnow.wpcomstaging.com/htdocs/wp-content/mcwhp/img/logo.svg"
-        },
```
Comment (viet-nv) [inline tokenLists/ethereum.tokenlist.json]: keep this

File: tokenLists/ethereum.tokenlist.json [hunk 2]
```diff
@@ -107,15 +95,6 @@
...
         },
-        {
-            "chainId": 1,
-            "address": "0x0bfEc35a1A3550Deed3F6fC76Dde7FC412729a91",
-            "_scan": "https://etherscan.io/token/0x0bfEc35a1A3550Deed3F6fC76Dde7FC412729a91",
-            "symbol": "XKNCA",
-            "name": "xKNCa",
-            "decimals": 18,
-            "logoURI": "https://assets.coingecko.com/coins/images/13995/thumb/Token-4.png?1613545763"
-        },
```
Comment (viet-nv) [inline tokenLists/ethereum.tokenlist.json]: keep

File: tokenLists/ethereum.tokenlist.json [hunk 3]
```diff
@@ -143,33 +122,6 @@
...
-        {
-            "chainId": 1,
-            "address": "0x53dfEa0A8CC2A2A2e425E1C174Bc162999723ea0",
-            "_scan": "https://etherscan.io/token/0x53dfEa0A8CC2A2A2e425E1C174Bc162999723ea0",
-            "symbol": "jCHF",
-            "name": "Jarvis Synthetic Swiss Franc",
-            "decimals": 18,
-            "logoURI": "https://i.imgur.com/Fp31dDB.png"
-        },
         {
```
Comment (viet-nv) [inline tokenLists/ethereum.tokenlist.json]: keep

File: tokenLists/ethereum.tokenlist.json [hunk 4]
```diff
@@ -231,15 +183,6 @@
...
         },
-        {
-            "chainId": 1,
-            "address": "0xb986F3a2d91d3704Dc974A24FB735dCc5E3C1E70",
-            "_scan": "https://etherscan.io/token/0xb986F3a2d91d3704Dc974A24FB735dCc5E3C1E70",
-            "symbol": "EUX",
-            "name": "dForce EUR",
-            "decimals": 18,
-            "logoURI": "https://raw.githubusercontent.com/dforce-network/documents/2eea27372fda6a207f60fba4274ad68008d63694/logos/Lending/EUX.svg"
-        },
```
Comment (viet-nv) [inline tokenLists/ethereum.tokenlist.json]: keep

File: tokenLists/ethereum.tokenlist.json [hunk 5]
```diff
@@ -330,15 +273,6 @@
...
         },
-        {
-            "chainId": 1,
-            "address": "0xD2af830E8CBdFed6CC11Bab697bB25496ed6FA62",
-            "_scan": "https://etherscan.io/token/0xD2af830E8CBdFed6CC11Bab697bB25496ed6FA62",
-            "symbol": "wOUSD",
-            "name": "Wrapped OUSD",
-            "decimals": 18,
-            "logoURI": "https://www.ousd.com/images/ousd-coin.svg"
-        },
```
Comment (viet-nv) [inline tokenLists/ethereum.tokenlist.json]: keep

File: tokenLists/matic.tokenlist.json [hunk 6]
```diff
@@ -79,51 +76,6 @@
...
-        },
-        {
-            "chainId": 137,
-            "address": "0xfAdE2934b8E7685070149034384fB7863860D86e",
-            "_scan": "https://polygonscan.com/token/0xfAdE2934b8E7685070149034384fB7863860D86e",
-            "symbol": "AUR-0112",
-            "name": "Aureus",
-            "decimals": 18,
-            "logoURI": "https://i.imgur.com/PyipL43.png"
-        },
```
Comment (viet-nv) [inline tokenLists/matic.tokenlist.json]: keep

File: tokenLists/matic.tokenlist.json [hunk 7]
```diff
@@ -142,77 +94,6 @@
...
-        {
-            "chainId": 137,
-            "address": "0xC25351811983818c9Fe6D8c580531819c8ADe90f",
-            "_scan": "https://polygonscan.com/token/0xC25351811983818c9Fe6D8c580531819c8ADe90f",
-            "symbol": "IDLE",
-            "name": "Idle (PoS)",
-            "decimals": 18,
-            "logoURI": "https://s2.coinmarketcap.com/static/img/coins/64x64/7841.png"
-        },
```
Comment (viet-nv) [inline tokenLists/matic.tokenlist.json]: keep

File: tokenLists/ethereum.tokenlist.json (hunk 1 above)
Comment (hungdoansy) [reply tokenLists/ethereum.tokenlist.json]: done

File: tokenLists/ethereum.tokenlist.json (hunk 3 above)
Comment (hungdoansy) [reply tokenLists/ethereum.tokenlist.json]: done

File: tokenLists/ethereum.tokenlist.json (hunk 4 above)
Comment (hungdoansy) [reply tokenLists/ethereum.tokenlist.json]: done

File: tokenLists/ethereum.tokenlist.json (hunk 2 above)
Comment (hungdoansy) [reply tokenLists/ethereum.tokenlist.json]: done

File: tokenLists/ethereum.tokenlist.json (hunk 5 above)
Comment (hungdoansy) [reply tokenLists/ethereum.tokenlist.json]: done

File: tokenLists/matic.tokenlist.json (hunk 6 above)
Comment (hungdoansy) [reply tokenLists/matic.tokenlist.json]: done

File: tokenLists/matic.tokenlist.json (hunk 7 above)
Comment (hungdoansy) [reply tokenLists/matic.tokenlist.json]: done

File: tokenLists/rinkeby.tokenlist.json [hunk 8]
```diff
@@ -67,12 +64,21 @@
...
+            "decimals": 18,
+            "logoURI": "ipfs://QmXttGpZrECX5qCyXbBQiqgQNytVGeZW5Anewvh2jc4psg"
+        },
+        {
+            "name": "Maker",
+            "address": "0xF9bA5210F91D0474bd1e1DcDAeC4C58E359AaD85",
+            "symbol": "MKR",
+            "decimals": 18,
+            "chainId": 4,
+            "logoURI": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/0xF9bA5210F91D0474bd1e1DcDAeC4C58E359AaD85/logo.png"
```
Comment (namgold) [inline tokenLists/rinkeby.tokenlist.json]: logo 404

**Review by namgold — APPROVED** [review]

File: tokenLists/rinkeby.tokenlist.json (hunk 8 above)
Comment (hungdoansy) [reply tokenLists/rinkeby.tokenlist.json]: fixed

File: tokenLists/ethereum.tokenlist.json [hunk 9]
```diff
@@ -384,7 +384,7 @@
             "name": "Band Protocol",
             "symbol": "BAND",
             "decimals": 18,
-            "logoURI": "https://assets.coingecko.com/coins/images/9545/thumb/band-protocol.png?1568730326"
+            "logoURI": "https://s2.coinmarketcap.com/static/img/coins/64x64/4679.png"
```
Comment (namgold) [inline tokenLists/ethereum.tokenlist.json:387]: https://s2.coinmarketcap.com/static/img/coins/128x128/4679.png

**Review by namgold — APPROVED** [review]

**Pull request merged**